e2e/util/context.go

```diff
@@ -60,6 +61,10 @@ func addToScheme(scheme *runtime.Scheme) error {
 		return err
 	}
 
+	if err := recipe.AddToScheme(scheme); err != nil {
+		return err
+	}
+
 	return ramen.AddToScheme(scheme)
 }
 
```

Comment:
Lets split the part adding recipe support to another commit or PR. This is preparation for adding recipes.